src/current/cockroachcloud/ccloud-faq.md

```diff
@@ -74,4 +70,4 @@ Refer to:
 
 ### How can we track and audit role-assignment actions in a CockroachDB {{ site.data.products.cloud }} organization?
 
-Any user with the Org Administrator role can access [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
\ No newline at end of file
+Any user with the Org Administrator role can access [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
```

Comment:
This sounded weird to me, but I don't know if what I have added fits with the sense either.
```suggestion
Any user with the Org Administrator role can access the [Cloud Organization audit logs]({% link cockroachcloud/cloud-org-audit-logs.md %}) capability to track when users are added and removed in the CockroachDB {{ site.data.products.cloud }} organization, and whenever any role assignment changes are performed for those users.
```

Reply:
Good idea, I tweaked your suggestion a bit to remove the "capability" indirection entirely.